src/gui/display.lua

```diff
@@ -129,7 +129,9 @@ end
 
 function Display:onMouseReleased(x, y, button, touch, presses)
 	if self.draggingWidget then
-		return self.draggingWidget:onMouseReleased(x, y, button, touch, presses)
+		local ret = self.draggingWidget:onMouseReleased(x, y, button, touch, presses)
+		self.draggingWidget = nil
+		return ret
```

Comment:
Maybe we should have an event called `onDragStop` (`onDragEnd`?) to let the widget decide if they want to stop the drag event.

Reply:
no